libraries/Microsoft.Bot.Builder.AI.LanguageGeneration/Analyzer.cs

```diff
@@ -4,24 +4,28 @@
 using System.Text.RegularExpressions;
 using Antlr4.Runtime.Misc;
 using Antlr4.Runtime.Tree;
-using Microsoft.Expressions;
+using Microsoft.Bot.Builder.Expressions;
+using Microsoft.Bot.Builder.Expressions.Parser;
 
 namespace Microsoft.Bot.Builder.AI.LanguageGeneration
 {
     public class Analyzer : LGFileParserBaseVisitor<List<string>>
     {
         public readonly EvaluationContext Context;
 
-        private Stack<EvaluationTarget> evalutationTargetStack = new Stack<EvaluationTarget>();
+        private IExpressionParser _expressionParser;
```

Comment:
Could this be private readonly? Doesn't seem to be re-assigned after construction.